ceph/ceph_admin.py

```diff
@@ -12,6 +12,17 @@
 logger = logging.getLogger(__name__)
 
 
+def nodes_is_empty(func):
+    def func_wrapper(*args, **kwargs):
+        if not kwargs.get("nodes"):
+            logger.warning("Nodes not found")
```

Comment:
IMO, we shouldn't warn as it is acceptable to have a configuration without some of the roles.